app/src/main/java/nerd/tuxmobil/fahrplan/congress/schedule/MainActivity.java

```diff
@@ -153,13 +154,25 @@ protected void onCreate(@Nullable Bundle savedInstanceState) {
         }
 
         Engagements.initUserEngagement(this);
+
+        onSessionAlarmNotificationTapped(getIntent());
     }
 
     @Override
     protected void onNewIntent(Intent intent) {
         super.onNewIntent(intent);
         MyApp.LogDebug(LOG_TAG, "onNewIntent");
         setIntent(intent);
+        if (intent != null) {
```

Comment:
I don't think this `null` check is necessary either.

Reply:
I mostly went defensive because in `FragmentActivity` the method signature is:

``` java
@Override
@CallSuper
protected void onNewIntent(@SuppressLint("UnknownNullness") Intent intent) { \* ... */ }
```

Let's see if being brave works out. :heavy_check_mark: Changed in 537674db6ea460607975e7b727b7f98d93ff78bc.